service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/EncounterMapper.java

```diff
@@ -30,9 +32,17 @@ public String mapEncounterToEhrComposition(Encounter encounter) {
             .effectiveTime(StatementTimeMappingUtils.prepareEffectiveTimeForEncounter(encounter))
             .availabilityTime(StatementTimeMappingUtils.prepareAvailabilityTimeForEncounter(encounter))
             .status(COMPLETE_CODE)
+            .type(buildType(encounter))
             .components(components);
 
         return TemplateUtils.fillTemplate(ENCOUNTER_STATEMENT_TO_EHR_COMPOSITION_TEMPLATE,
             encounterStatementTemplateParameters.build());
     }
+
+    private String buildType(Encounter encounter) {
```

Comment:
Encounter.type is a special case handled by NIAD-1113 and should have been omitted by 1087. I'll update the story and these changed to EncounterMapper can be rolled back.